### PR TITLE
Clean up issue templates

### PR DESCRIPTION
**Pull request by driazati**
* Cleans up labels and makes them consistent, removes them from issue titles in favor of labels on the PR
* Adds a "feature request" and "other" issue template
* Removes the "CI problem" template as it's mostly covered by the "flaky test" template
* Removes the language about closing unresolved stale issues

cc @areusch @denise-k @hpanda-naut

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/06-other.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "\U00002753 Other"
+about: File an issue that doesn't fit into the other categories
+title: ""
+---
```
Comment (areusch) [inline .github/ISSUE_TEMPLATE/06-other.md:5]: should we add a needs-triage label?

**Review by areusch — COMMENTED** [review]
left a couple comments

File: .github/ISSUE_TEMPLATE/06-other.md (hunk 1 above)
Comment (driazati) [reply .github/ISSUE_TEMPLATE/06-other.md:5]: i was thinking it should go the other way and we assign a `triaged` label once issues have been sent to the relevant people, that way nothing is missed

File: .github/ISSUE_TEMPLATE/02-feature-request.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
```
[the remark this answers is not in the document]
Comment (driazati) [reply .github/ISSUE_TEMPLATE/02-feature-request.md:1]: did a quick write up here for discussion: https://discuss.tvm.apache.org/t/consolidate-development-activity-on-github/12219

File: .github/ISSUE_TEMPLATE/06-other.md (hunk 1 above)
Comment (areusch) [reply .github/ISSUE_TEMPLATE/06-other.md:5]: i support that!

**Comment by driazati** [comment]
Moving to a draft until we can align better on https://discuss.tvm.apache.org/t/consolidate-development-activity-on-github/12219

**Pull request closed without merging**